tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from blockbuster import blockbuster_ctx
+
+
+@pytest.fixture(autouse=True)
+def blockbuster(request):
+    with blockbuster_ctx("fastapi") as bb:
+        bb.functions["io.BufferedReader.read"].can_block_in(
+            "starlette/testclient.py", "receive"
+        )
+        for func in [
+            "os.stat",
+            "io.TextIOWrapper.read",
+        ]:
+            bb.functions[func].can_block_in(
+                "pydantic/networks.py", "import_email_validator"
```

Comment:
Pydantic's import_email_validator calls metadata.version which does a blocking read.
The result is cached so the impact of blocking the event loop is not big.